Kernel/Net/ICMP.h

```diff
@@ -7,9 +7,9 @@
 #pragma once
 
 #include <AK/MACAddress.h>
-#include <Kernel/Net/IPv4.h>
+#include <Kernel/Net/IPv4/IPv4.h>
 
-struct ICMPType {
+struct ICMPv4Type {
```

Comment:
It's not only files then, it's also about renaming things.
I'd suggest changing the commit title to something like this:
```
Kernel/Net: Rename IPv4-related files and structs with appropriate prefix
```

Reply:
Done